src/esp/metadata/managers/PhysicsAttributesManager.cpp

```diff
@@ -68,28 +68,6 @@ void PhysicsAttributesManager::setValsFromJSONDoc(
       std::bind(&PhysicsManagerAttributes::setGravity, physicsManagerAttributes,
                 _1));
 
-  // load the rigid object library metadata (no physics init yet...)
-  if (jsonConfig.HasMember("rigid object paths") &&
-      jsonConfig["rigid object paths"].IsArray()) {
-    std::string configDirectory = physicsManagerAttributes->getFileDirectory();
-
-    const auto& paths = jsonConfig["rigid object paths"];
-    for (rapidjson::SizeType i = 0; i < paths.Size(); i++) {
-      if (!paths[i].IsString()) {
-        LOG(ERROR) << "PhysicsAttributesManager::createAttributesTemplate "
-                      ":Invalid value in physics scene config -rigid object "
-                      "library- array "
-                   << i;
-        continue;
-      }
-
-      std::string absolutePath =
-          Cr::Utility::Directory::join(configDirectory, paths[i].GetString());
-      // load all object templates available as configs in absolutePath
-      objectAttributesMgr_->loadAllConfigsFromPath(absolutePath, true);
-    }
-  }  // if load rigid object library metadata
-
```

Comment:
This was removed because, with the new dataset-driven system, it does not make sense for object templates to be loaded "globally" via the physics manager.